iOS/DuckDuckGo/DuckPlayer/NativeUI/Views/DuckPlayerViewModel.swift

```diff
@@ -169,7 +172,9 @@ final class DuckPlayerViewModel: ObservableObject {
         var components = URLComponents(url: videoURL, resolvingAgainstBaseURL: true)
         let seconds = Int(timestamp)
         var queryItems = components?.queryItems ?? []
-        queryItems.append(URLQueryItem(name: Constants.startParameter, value: String(seconds)))
+        if seconds >= 5 {
+            queryItems.append(URLQueryItem(name: Constants.startParameter, value: String(seconds)))
+        }
```

Comment:
Just so I understand: what youtube bug does this prevent? I haven't found anything on it in the Asana link.

Reply:
The new YouTube player doesn't show a loading indicator when you include the "start" parameter. This makes the player go blank before the video loads, which sucks.

Since most users open DuckPlayer immediately (or it opens automatically), we skip the parameter if the playhead is under 5 seconds. This makes YouTube show the loading indicator properly.

This isn't really a bug - it's a workaround for YouTube's player being broken.